presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -86,6 +87,11 @@
 
     // Requires Pinot version >= 0.4.0.
     private boolean usePinotSqlForBrokerQueries = true;
+    // Requires Pinot version >= 0.5.0.
+    private boolean useStreamingForSegmentQueries;
+    private boolean usePlainTextForStreamingServer = true;
+    private int serverGrpcPort = 8090;
```

Comment:
Please add a comment here, that this is only used as a fallback if the controller does not provide a per server grpc port (if a pinot version older than XX) is running.

Also, should the default be something like -1 ... to denote that an RPC port fall back is not set ? I think 8090 encodes some pinot information here that ideally belongs in some config file.

Reply:
True, will just put -1 here, which means unless been set, this tis not used.